### PR TITLE
Tuneup malware report

### PR DESCRIPTION
**Pull request by ghost**
Added malware report (#22375) only relevant if `malware` `caveat` is accepted (#22383)

### CODE REVIEW

**Comment by adidalal** [comment]
Should be of the form

```
caveats do
    malware(radar_number)
end
```

(caveats go under app/binary stanzas)

File: Casks/tuneup.rb [hunk 1]
```diff
@@ -11,4 +11,7 @@
   license :commercial
 
   installer manual: 'TuneUp-Installer.app'
+  caveats do
+    malware(27013246)
```
Comment (vitorgalvao) [inline Casks/tuneup.rb]: Do instead `malware '27013246'`. It’s consistent with other `caveats`, and actually sets it as a String (which we want).

File: Casks/tuneup.rb [hunk 2]
```diff
@@ -11,4 +11,7 @@
   license :commercial
 
   installer manual: 'TuneUp-Installer.app'
+  caveats do
```
Comment (vitorgalvao) [inline Casks/tuneup.rb]: Needs a space before this, for `audit` to pass.

Comment (vitorgalvao) [inline Casks/tuneup.rb]: Needs an empty line space before this, for `audit` to pass.

**Comment by decorate** [comment]
Thank you @vitorgalvao I have made the recommended changes.

**Comment by vitorgalvao** [comment]
Thank you. Now on hold until we finish discussing and merge https://github.com/caskroom/homebrew-cask/pull/22383.

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

In this case, lack of running the `style` command is breaking the build.

**Comment by vitorgalvao** [comment]
@decorate This just needed a simple fix to correct the failing build, and for us to have the record of the template. Will you submit again?

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8ab0616293eb9128a46e478afdb726a0aa8b811b. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**